scripts/chocolatey-packages.bat

```diff
@@ -5,6 +5,8 @@ set choco_binary=c:\programdata\chocolatey\bin\choco.exe
 %choco_binary% install sysinternals -y
 %choco_binary% install curl -y
 %choco_binary% install jre8 -y
+%choco_binary% install maven -y
```

Comment:
how much disk space does the maven and jdk8 need? right now the base image is 12G.